### ISSUE TITLE
[Request] Add rating to Info Wall View

### ISSUE THREAD

**Issue by Rouzax** [jurialmunkey/skin.arctic.zephyr.2#88]
Running the latest version from the repo.
On the Info Wall view with movies, it does not show any rating.
It's the screen with on the left hand 8 movies posters, right top the fanart and bottom right the title, run time and plot.
I do have the additional config for the rottentomatoes and IMDb score which does show in other views.
Would it be possible to add it?

**Comment by quantumwebco** [comment]
I added the star ratings just at the bottom so I could see them when browsing for myself. Feel free to use the updated file.

@jurialmunkey I am thinking to do some more work on this view to add genres, ratings, next air date etc. Shall I create a PR when I've updated it? Are you accepting contributors? 

[Includes_View_51_Wall.zip](https://github.com/jurialmunkey/skin.arctic.zephyr.2/files/3587884/Includes_View_51_Wall.zip)

**Comment by quantumwebco** [comment]
I added the star ratings just at the bottom so I could see them when browsing for myself. Feel free to use the updated file.

@jurialmunkey I am thinking to do some more work on this view to add genres, ratings, next air date etc. Shall I create a PR when I've updated it? Are you accepting contributors?

[Includes_View_52_Showcase.zip](https://github.com/jurialmunkey/skin.arctic.zephyr.2/files/3587920/Includes_View_52_Showcase.zip)
